### ISSUE TITLE
Feature request: allow deleting single connections to dive computers

### ISSUE THREAD

**Issue by ghost** [subsurface/subsurface#2162]
### Describe the issue:

- [ ] Bug
- [ ] Change request
- [x ] New feature request
- [ ] Discussion request

### Issue long description:
I added a Shearwater Perdix AI to my equipment and had to experiment with the Bluetooth connection (trying internal Bluetooth and the dongle Shearwater sells with the Perdix AI, than switching back to my OSTC) and now I have multiple entries for each computer. When I want to read out dive data I never know which button to click since the connections a not distinguishable by name.

![download_dialog](https://user-images.githubusercontent.com/3070848/60754605-c00d4b00-9fe3-11e9-9882-12546f1bbd20.png)

### Operating system:

OSTC sport, Shearwater Perdix AI

### Subsurface version:

4.8.6 official

### Steps to reproduce:

1) open download dialog
2) add one or more new connections
3) get confused which connection worked and which did not work since the connections to the same computer a not distinguishable by name
4) find no way to delete a single connection
5) delete all connections using "Preferences" -> "General" -> "Clear settings" -> "Reset remembered dive computers"
6) start over

### Current behavior:

The only option is to delete all devices.

### Expected behavior:

Being able to select one or more devices and delete them while leaving other untouched, preferrably directly in the dialog.

**Comment by dirkhh** [comment]
Good idea. We're working on wrapping up 4.9 right now and as this would change texts (and we are in the final stages of translation) this won't make it into that release, but it should be fairly easy to add for a release after that.

**Comment by ghost** [comment]
eliminated typo
